float32 set_velocity
float32 set_degree    #-28 ~ 28
float32 set_brake     #1 ~ 150
int8    set_gear      #foward = 0, N = 1, backward = 2
